web/src/connectors/task-annotator-connector/utils.ts

```diff
@@ -0,0 +1,129 @@
+import { CategoryDataAttrType } from '../../api/typings';
+import { PageSize } from '../../shared/components/document-pages/document-pages';
+
+interface Token {
```

Comment:
I think it would be better to have separate module like `types` and store our types there

Reply:
We already had types for this code, but it looks a little confusing and incorrect to some of the real BE responses.

  I think i can transfer new models to types.ts in current folder like

Reply:
fixed https://github.com/epam/badgerdoc/pull/584/commits/ab482c315552723f0515f8bf935d1af228cadb30